config/config.toml.example

```diff
@@ -446,3 +446,7 @@ allow-expression-index = false
 [isolation-read]
 # engines means allow the tidb server read data from which types of engines. options: "tikv", "tiflash", "tidb".
 engines = ["tikv", "tiflash", "tidb"]
+
+# set labels for the tidb server
```

Comment:
please add some use cases for this config. do not make users confused.

Reply:
updated.